iothub/device/tests/RetryPolicies/IotHubClientExponentialBackoffRetryPolicyTests.cs

```diff
@@ -2,15 +2,13 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
-using System.Threading;
 using FluentAssertions;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
 namespace Microsoft.Azure.Devices.Client.Test
 {
     [TestClass]
     [TestCategory("Unit")]
-    [TestCategory("IoTHub")]
```

Comment:
I don't see the advantage of tagging unit tests as IoTHub. They have minimal running time and don't provide any substantial benefit if we exclude them from our runs.

Reply:
The goal was to separate hub from dps tests.